pandas/core/generic.py

```diff
@@ -4363,111 +4288,8 @@ def _slice(self, slobj: slice, axis: AxisInt = 0) -> Self:
-        result._set_is_copy(self, copy=is_copy)
         return result
 
-    @final
-    def _set_is_copy(self, ref: NDFrame, copy: bool_t = True) -> None:
-        if not copy:
-            self._is_copy = None
-        else:
-            assert ref is not None
-            self._is_copy = weakref.ref(ref)
-
-    def _check_is_chained_assignment_possible(self) -> bool_t:
-        """
-        Check if we are a view, have a cacher, and are of mixed type.
-        If so, then force a setitem_copy check.
-
-        Should be called just near setting a value
-
-        Will return a boolean if it we are a view and are cached, but a
-        single-dtype meaning that the cacher should be updated following
-        setting.
-        """
-        if self._is_copy:
-            self._check_setitem_copy(t="referent")
-        return False
-
-    @final
-    def _check_setitem_copy(self, t: str = "setting", force: bool_t = False):
-        """
-
-        Parameters
-        ----------
-        t : str, the type of setting error
-        force : bool, default False
-           If True, then force showing an error.
-
-        validate if we are doing a setitem on a chained copy.
-
-        It is technically possible to figure out that we are setting on
-        a copy even WITH a multi-dtyped pandas object. In other words, some
-        blocks may be views while other are not. Currently _is_view will ALWAYS
-        return False for multi-blocks to avoid having to handle this case.
-
-        df = DataFrame(np.arange(0,9), columns=['count'])
-        df['group'] = 'b'
-
-        # This technically need not raise SettingWithCopy if both are view
-        # (which is not generally guaranteed but is usually True.  However,
-        # this is in general not a good practice and we recommend using .loc.
-        df.iloc[0:5]['group'] = 'a'
-
-        """
-        if using_copy_on_write() or warn_copy_on_write():
-            return
-
-        # return early if the check is not needed
-        if not (force or self._is_copy):
-            return
-
-        value = config.get_option("mode.chained_assignment")
-        if value is None:
-            return
-
-        # see if the copy is not actually referred; if so, then dissolve
-        # the copy weakref
-        if self._is_copy is not None and not isinstance(self._is_copy, str):
-            r = self._is_copy()
-            if not gc.get_referents(r) or (r is not None and r.shape == self.shape):
-                self._is_copy = None
-                return
-
-        # a custom message
-        if isinstance(self._is_copy, str):
-            t = self._is_copy
-
-        elif t == "referent":
-            t = (
-                "\n"
-                "A value is trying to be set on a copy of a slice from a "
-                "DataFrame\n\n"
-                "See the caveats in the documentation: "
-                "https://pandas.pydata.org/pandas-docs/stable/user_guide/"
```

Comment:
are these docs still there?

Reply:
Yeah I’ll continue this pr later or
Tomorrow, the code things should be gone, docs not yet